Nodejs/Product/Nodejs/NpmUI/NpmPackageInstallWindow.xaml

```diff
@@ -343,7 +344,7 @@
                               Content="{Binding Path=SelectedPackage}"
                               ContentTemplate="{StaticResource PackageInfoTemplate}"
                               IsTabStop="True" AutomationProperties.Name="{Binding Path=SelectedPackage.Name}"
-                              TabIndex="3" />
+                              Focusable="False"/>
```

Comment:
What is the difference between `IsTabStop` (just above) and `Focusable` here?

Also, is the general intent to not stop on the vertical scrollbar while tabbing?

Reply:
I didn't consider the scrolling! I don't know the answer for that, I'll follow up with the accessibility folks. 

In regards to `IsTabStop` and `Focusable` seems like the first is the right solution if we don't want to stop on the control. Depending on their answer about the scrolling, I'll fix this.

Reply:
Based on discussion with the accessibility experts there should be no change here. Ideally it should be only focusable when the scroll is visible, but since this is the default behavior of the control and changing it seems to be a ton of effort, for now, we should leave it as is.

Rollback the changes.